community-membership.md

```diff
@@ -61,7 +61,7 @@ remain active contributors to the community.
     - Contributing to SIG, subproject, or community discussions (e.g. meetings,
       Slack, email discussion forums)
 - Subscribed to [dev@kubernetes.io]
-- Have read the [contributor guide]
+- Have read the [contributor guide]()
```

Comment:
Also, this is not necessary :)

Reply:
I see that now, my bad.  Thanks @ArvindParekh!